script.js

```diff
@@ -912,21 +907,30 @@ require([
   //// Zoom to Township/Section/Range Feature ////
   ////////////////////////////////////////////////
 
-  var townshipSelect = dom.byId("selectTownship");
-  var rangeSelect = dom.byId("selectRange");
-  var sectionSelect = dom.byId("selectSection");
+  const townshipSelect = dom.byId("selectTownship");
+  const rangeSelect = dom.byId("selectRange");
+  const sectionSelect = dom.byId("selectSection");
 
   // when mapView is ready, build the first dropdown for township selection
   mapView.when(async function () {
-    var townshipQuery = new Query({
+    const townshipQuery = new Query({
       where: "tdir <> ' ' AND NOT (CAST(twn_ch AS int) > '8' AND tdir = 'N')",
       outFields: ["twn_ch", "tdir"],
       returnDistinctValues: true,
-      orderByFields: ["twn_ch", "tdir"],
+      orderByFields: ["twn_ch", "tdir"]
     });
+
+    const rangeQuery = new Query({
+      where: "rdir <> ' '",
+      outFields: ["rng_ch", "rdir"],
+      returnDistinctValues: true,
+      orderByFields: ["rng_ch", "rdir"]
+    })
```

Comment:
Is it possible to combine this query with the previous one?

Reply:
I can, but I feel like it would be at the expense of some clarity. They are similar but the select for townships is a little bit more complicated